src/astgen.zig

```diff
@@ -2441,10 +2457,120 @@ fn nodeMayNeedMemoryLocation(start_node: *ast.Node) bool {
+                    .{ "@bitReverse", false },
+                    .{ "@byteOffsetOf", false },
+                    .{ "@call", true },
+                    .{ "@cDefine", false },
+                    .{ "@cImport", false },
+                    .{ "@cInclude", false },
+                    .{ "@clz", false },
+                    .{ "@cmpxchgStrong", false },
+                    .{ "@cmpxchgWeak", false },
+                    .{ "@compileError", false },
+                    .{ "@compileLog", false },
+                    .{ "@ctz", false },
+                    .{ "@cUndef", false },
+                    .{ "@divExact", false },
+                    .{ "@divFloor", false },
+                    .{ "@divTrunc", false },
+                    .{ "@embedFile", false },
+                    .{ "@enumToInt", false },
+                    .{ "@errorName", false },
+                    .{ "@errorReturnTrace", false },
+                    .{ "@errorToInt", false },
+                    .{ "@errSetCast", false },
+                    .{ "@export", false },
+                    .{ "@fence", false },
+                    .{ "@field", true },
+                    .{ "@fieldParentPtr", false },
+                    .{ "@floatCast", false },
+                    .{ "@floatToInt", false },
+                    .{ "@frame", false },
+                    .{ "@Frame", false },
+                    .{ "@frameAddress", false },
+                    .{ "@frameSize", false },
+                    .{ "@hasDecl", false },
+                    .{ "@hasField", false },
+                    .{ "@import", false },
+                    .{ "@intCast", false },
+                    .{ "@intToEnum", false },
+                    .{ "@intToError", false },
+                    .{ "@intToFloat", false },
+                    .{ "@intToPtr", false },
+                    .{ "@memcpy", false },
+                    .{ "@memset", false },
+                    .{ "@wasmMemorySize", false },
+                    .{ "@wasmMemoryGrow", false },
+                    .{ "@mod", false },
+                    .{ "@mulWithOverflow", false },
+                    .{ "@panic", false },
+                    .{ "@popCount", false },
+                    .{ "@ptrCast", false },
+                    .{ "@ptrToInt", false },
+                    .{ "@rem", false },
+                    .{ "@returnAddress", false },
+                    .{ "@setAlignStack", false },
+                    .{ "@setCold", false },
+                    .{ "@setEvalBranchQuota", false },
+                    .{ "@setFloatMode", false },
+                    .{ "@setRuntimeSafety", false },
+                    .{ "@shlExact", false },
+                    .{ "@shlWithOverflow", false },
+                    .{ "@shrExact", false },
+                    .{ "@shuffle", false },
+                    .{ "@sizeOf", false },
+                    .{ "@splat", true },
+                    .{ "@reduce", true },
```

Comment:
false.

Reply:
Is splat also false, or only reduce?

Reply:
This was only meant for reduce, splat and shuffle return vectors and I'm not sure if those use result locations or not.

Reply:
The other two reviews were also only for the last line, not sure why github is showing four lines for them.